src/App.tsx

```diff
@@ -16,141 +85,43 @@ export const App: React.FC = () => {
       <div className="todoapp__content">
         <header className="todoapp__header">
           {/* this buttons is active only if there are some active todos */}
+          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
           <button type="button" className="todoapp__toggle-all active" />
 
           {/* Add a todo on form submit */}
-          <form>
+          <form
+            onSubmit={e => {
+              e.preventDefault();
+              const newTodoTitle = (e.target as HTMLFormElement)['new-todo']
+                .value;
+
+              handleCreateTodo(newTodoTitle);
+              (e.target as HTMLFormElement).reset();
```

Comment:
It would be better to use separate function and use this separate function in form.